addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -106,6 +129,7 @@ export class LinkPlugin extends Plugin {
                 isAvailable: () => this.shared.getEditableSelection().inEditable,
             }
         );
+        this.renderingClasses = new Set(this.resources["history_rendering_classes"]);
```

Comment:
@Goaman Should this resource be renamed without its "history" prefix?

Reply:
@cammarosano I'm fine either way